src/go/k8s/controllers/redpanda/console_controller.go

```diff
@@ -265,6 +271,11 @@ func (r *ConsoleReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&appsv1.Deployment{}).
 		Owns(&corev1.Service{}).
 		Owns(&netv1.Ingress{}).
+		Watches(
+			&source.Kind{Type: &redpandav1alpha1.Cluster{}},
+			handler.EnqueueRequestsFromMapFunc(r.reconcileConsoleForCluster),
+			builder.WithPredicates(predicate.ResourceVersionChangedPredicate{}),
```

Comment:
I'm also thinking that if right now we are only interested with create events, should we only watch those? Otherwise console will reconcile with unnecessary events e.g. update wdyt?

Reply:
For the requeues, I think the fact that the cluster is ready does not automatically imply that the internal APIs are available and also the initial request can fail, so it should be retried.

I think it would be also good if a Console reconcile event generated by a Cluster change can reset the exponential backoff counter, so that in case of errors in calling the APIs, retries have a shorter delay. But I'm not 100% sure if this will happen.

Reply:
I think atm we are requeueing the request if cluster is not yet configured, maybe we don't need this anymore as we are watching cluster events? :thinking: 
In my mind:
1. console reconciles
2. cluster is not yet configured, do nothing no requeue
3. cluster is now configured, sets "status.clusterConfigured=true"
4. this will be a resourceVersion change and should trigger a console reconcile